hack/build.sh

```diff
@@ -88,7 +75,7 @@ for dir in ${dirs}; do
     docker_build_with_version Dockerfile
 #  fi
 
-  if [[ -v TEST_MODE ]]; then
+  if [ ! -z $TEST_MODE ]; then
```

Comment:
You can restore the double [[ (I think).
Also, I think that a more correct check would be:
```
if [[ -z "${TEST_MODE:-}" ]]; then
```